reflex.cabal

```diff
@@ -46,6 +46,7 @@ library
     containers >= 0.5 && < 0.7,
     data-default >= 0.5 && < 0.8,
     dependent-map >= 0.2.4 && < 0.3,
+    distributive,
```

Comment:
Should be safe since our lower bound of `semigroupoids` already depends on `distributive`